### PR TITLE
You are conflating your Layouts and your templates

### PR DESCRIPTION
**Pull request by joshchernoff**
This this string off commits I address each issue one by one as I read over the error reported. 

Nothing about any of these errors are a direct result of following the guides. 

All of the issues observed are the result of conflating the layout with the template. 

My advice. Take more time to understand the errors reported as they directly state the issue at hand.

### CODE REVIEW

**Comment by joshchernoff** [comment]
Screen Shot 2020-05-07 at 1 12 31 PM
Screen Shot 2020-05-07 at 1 12 38 PM

File: lib/bamboo_example/email.ex [hunk 1]
```diff
@@ -5,6 +5,8 @@ defmodule BambooExample.Email do
   # as per https://hexdocs.pm/bamboo/Bamboo.Phoenix.html
   def welcome_email do
     new_email()
+    |> from({"Some one random", "no-reply@exmaple.com"})
```
Comment (tmpduarte) [inline lib/bamboo_example/email.ex:8]: @joshchernoff this I knew would happen. I was just trying to reproduce the render issue I was having

File: config/config.exs [hunk 2]
```diff
@@ -18,7 +18,9 @@ config :bamboo_example, BambooExampleWeb.Endpoint,
   pubsub_server: BambooExample.PubSub,
   live_view: [signing_salt: "iuC1odaa"]
 
-config :bamboo_example, BambooExample.Mailer, adapter: Bamboo.LocalAdapter
+config :bamboo_example, BambooExample.Mailer,
+  adapter: Bamboo.LocalAdapter,
+  open_email_in_browser_url: "http://localhost:4000/sent_emails"
```
Comment (tmpduarte) [inline config/config.exs:23]: didn't knew about this 🤔

**Comment by tmpduarte** [comment]
👍 indeed I confused the templates with the email itself

File: lib/bamboo_example/email.ex (hunk 1 above)
Comment (joshchernoff) [reply lib/bamboo_example/email.ex:8]: Sure but the point I was trying to make is to follow the errors as they will guide you threw the issue

**Pull request merged**